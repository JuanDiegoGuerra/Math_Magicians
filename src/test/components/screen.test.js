import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Screen from '../../components/screen';

describe('The Screen component', () => {
  it('Renders correctly', () => {
    const tree = renderer.create(<Screen render="0" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Displays the correct value', () => {
    render(<Screen render="9" />);
    expect(screen.getByTestId('screen-element')).toHaveTextContent('9');
  });
});
