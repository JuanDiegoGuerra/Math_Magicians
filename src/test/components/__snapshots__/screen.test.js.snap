// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Screen component Renders correctly 1`] = `
<div
  className="calc-number calc-screen"
  data-testid="screen-element"
>
   
  0
   
</div>
`;
